binrz/man/rz-find.1

```diff
@@ -1,66 +1,70 @@
-.Dd Jun 24, 2020
+.Dd Jan 24, 2024
 .Dt RZ_FIND 1
 .Sh NAME
 .Nm rz-find
 .Nd Advanced commandline hexadecimal editor
 .Sh SYNOPSIS
 .Nm rz-find
-.Op Fl ijzZXnrhqv
+.Op Fl hijmnrqvxzZ
+.Op Fl a Ar align
 .Op Fl b Ar size
-.Op Fl f Ar from
-.Op Fl F Ar file
-.Op Fl t Ar to
-.Op Fl [m|s|e] Ar str
+.Op Fl f/t Ar from/to
+.Op Fl  Ar [e|s|w|S|I] str
 .Op Fl x Ar hex
-.Ar file|dir
+.Op Fl  Ar |file|dir ..
 .Sh DESCRIPTION
-rz-find is a program to find byte patterns into files
+.Nm rz-find
```

Comment:
Just like with `rz-ax,` please add a note that its part of the Rizin project. For this and all other commands.

Reply:
there is no description in rz-bin should i add one like this 
![image](https://github.com/rizinorg/rizin/assets/62786136/a1a8cfbf-0546-49c5-930e-30d9bb0b6864)

Reply:
there is no description in rz-bin should i add one like this 
![image](https://github.com/rizinorg/rizin/assets/62786136/b076f7af-6c7e-4214-8d8e-67f17cf8d746)

Reply:
Yes